src/m365/aad/commands/group/group-list.ts

```diff
@@ -0,0 +1,60 @@
+import * as chalk from 'chalk';
+import { Logger } from '../../../../cli';
+import {
+  CommandOption
+} from '../../../../Command';
+import GlobalOptions from '../../../../GlobalOptions';
+import { GraphItemsListCommand } from '../../../base/GraphItemsListCommand';
+import commands from '../../commands';
+import { Group } from './Group';
+
+interface CommandArgs {
+  options: Options;
+}
+
+interface Options extends GlobalOptions {
+  output?: string;
```

Comment:
`output` is already defined on `GlobalOptions` and we shouldn't redefine it here.